src/graphql/schemas/ContactSchema.ts

```diff
@@ -0,0 +1,47 @@
+const schema = `
+  extend type Mutation {
+    mutateContact(request: MutateContactRequest!): Contact
+    deleteContactsByIds(request: DeleteContactsByIdsRequest!): [Contact]
+  }
+
+  extend type Query {
+    getContactsWithOptions(request: GetContactsRequest): GetContactsResponse
+  }
+
+  input GetContactsRequest {
+    options: QueryOptions
+    filters: ContactFilters
+  }
+
+  type GetContactsResponse {
+    contacts: [Contact]
+  }
+  
+  input DeleteContactsByIdsRequest {
+    ids: [ID]!
+  }
+
+  input ContactFilters {
+    contactId: ID!
+  }
+
+  input MutateContactRequest {
+    id: String
+    userId: String
```

Comment:
userId comes from the token,shouldnt be an input -> important as the user cannot update any data which doesnt belong to him